concept/type/attributetype.feature

```diff
@@ -161,6 +161,10 @@ Feature: Concept Attribute Type
     When session opens transaction of type: write
     Then attribute(email) is abstract: true
     Then attribute(email) as(string) put: alice@email.com; throws exception
+    When session opens transaction of type: write
+    When put attribute type: company-email, with value type: string
+    When attribute(company-email) set supertype: email
+    Then attribute(email) set abstract: false; throws exception
```

Comment:
we can create a concrete child attribute type of an abstract parent attribute type, but can no longer change the parent to be non-abstract

Reply:
Yeah this is fair enough, I suspect we already have a similar test for TypeQL in `define.feature` but overlap is inevitable